Week2/homework/maartjes_work.js

```diff
@@ -44,4 +44,21 @@ const tuesday = [
 
 const tasks = monday.concat(tuesday);
 
-// Add your code here
+//Map the tasks to durations in hours----
+const minutesToHours = tasks.map(tasks => tasks.duration / 60);
```

Comment:
you are also doing this on line 58 and I think you don't use this variable somewhere else. So, I don't see any need to include it.